include/secure_services.h

```diff
@@ -52,6 +52,20 @@ void spm_request_system_reboot(void);
 int spm_request_random_number(u8_t *output, size_t len, size_t *olen);
 
 
+/** Request a read operation to be executed from Secure Firmware.
+ *
+ * @param[out] destination Pointer to destination array where the content is
+ *                         to be copied.
+ * @param[in]  source      Pointer to the source of data to be copied.
+ * @param[in]  num         Number of bytes to copy.
```

Comment:
nit: I suggest `len` here.

Reply:
Since this as a wrapper around `memcpy` I used the same names for the arguments (destination, source, num).

Reply:
changed to len since it is used elsewhere in this file :)